### PR TITLE
Wayland-drm: Fix not finding wl_drm_interface on systems with libglvnd

### PR DESCRIPTION
**Pull request by jwrdegoede**
We do not want just any libEGL.so.1 we want mesa's libEGL.so.1 as that
is the only way which defines the wl_drm_interface symbol we need,
one systems with libglvnd libEGL.so.1 is a dispatcher library provided
by libglvnd and the actual mesa libEGL we want is named libEGL_mesa.so.0
so try that first.

Signed-off-by: Hans de Goede <hdegoede@redhat.com>

### CODE REVIEW

File: va/wayland/va_wayland_drm.c [hunk 1]
```diff
@@ -38,7 +38,10 @@
 #include "wayland-drm-client-protocol.h"
 
 /* XXX: Wayland/DRM support currently lives in Mesa libEGL.so.* library */
-#define LIBWAYLAND_DRM_NAME "libEGL.so.1"
```
Comment (seanvk) [inline va/wayland/va_wayland_drm.c]: Good point, I agree.

Comment (seanvk) [inline va/wayland/va_wayland_drm.c]: We need to make similar changes to 01org/libva-utils

**Comment by jwrdegoede** [comment]
Hi,

On 15-02-17 18:43, Sean V. Kelley wrote:
> *@seanvk* commented on this pull request.
>
> ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
>
> In va/wayland/va_wayland_drm.c <https://github.com/01org/libva/pull/27>:
>
>> @@ -38,7 +38,10 @@
>  #include "wayland-drm-client-protocol.h"
>
>  /* XXX: Wayland/DRM support currently lives in Mesa libEGL.so.* library */
> -#define LIBWAYLAND_DRM_NAME "libEGL.so.1"
>
> We need to make similar changes to 01org/libva-utils

[hans@shalem projects]$ grep -r libEGL libva-utils
[hans@shalem projects]$

So that does not seem to be necessary.

Regards,

Hans

**Comment by seanvk** [comment]
It's a potential dep in the future because if we write specific EGL based tests.  This is a result of splitting libva-utils from libva.

**Comment by seanvk** [comment]
But you are right, we don't need it for libva-utils at this time. @jwrdegoede

**Comment by seanvk** [comment]
I actually removed the EGL deps from libva-utils configure.ac recently

**Pull request merged**